operators/tokenizer/bpe_kernels.h

```diff
@@ -115,7 +115,7 @@ class JsonFastTokenizer : KernelBpeTokenizer {
                      std::optional<ortc::Tensor<int64_t>*> offset_mapping) const;
 
  public:
-  auto GetAddedTokens() const { return added_tokens_; }
+  auto& GetAddedTokens() const { return added_tokens_; }
```

Comment:
Is it possible to share more detail what kind of C++ compiler or flag are triggering error? We can add it in ort-extensions to avoid this kind of issue in the future.

Reply:
I am using linux GCC compiler 12.x for this. I think this would be an error regardless of the compiler flags being used. Not sure what the best way it to catch this error.

Reply:
Thanks, it looks like only GCC 12 catch this error, and the pipeline are only using

 -- The CXX compiler identification is GNU 9.4.0